kolibri/plugins/learn/assets/src/views/content-card-group-carousel/index.vue

```diff
@@ -20,7 +20,7 @@
           v-show="!isLastSet"
           :disabled="isLastSet"
           :disableRipple="true"
-          icon="arrow_forward"
+          :icon="isRtl ? 'arrow_back' : 'arrow_forward'"
```

Comment:
see @christianmemije's work for some related changes: https://github.com/learningequality/kolibri/pull/2702

In particular, we may want to use his new `rtl-icon` class to flip the arrows rather than choosing a different icon, since semantically the action is still "forward" even if it's pointing left.

https://github.com/learningequality/kolibri/pull/2702/files#diff-af075b31abd74f9069c394564ab5b4ebR144